docs/metrics.md

```diff
@@ -28,6 +28,69 @@ openstack-cloud-controller-manager
   --secure-port=10258
 ```
 
+### Exposing metrics to prometheus operator
+
+Default setup assumes that cloud-controller-manager is located in master.
+Please note that firewall between prometheus (usually normal node) and master port 10258 needs to be open.
+After that you can just apply following manifests and you should be able to see statistics in prometheus soon.
+
+```yaml
```

Comment:
do you think it's reasonable to put those configuration files in `sample` so that we can update it easily later on and link from here ?

Reply:
I do not think so because prometheus operator is not installed in all clusters

Reply:
ok, thanks for the info